cmeutils/plotting.py

```diff
@@ -29,9 +29,13 @@ def get_histogram(data, normalize=False, bins="auto", x_range=None):
         Array of the bin height values
 
     """
-    bin_heights, bin_borders = np.histogram(data, bins=bins, range=x_range)
-    if normalize is True:
-        bin_heights = bin_heights / sum(bin_heights)
+    if normalize:
+        density = True
+    else:
+        density = False
+    bin_heights, bin_borders = np.histogram(
+        a=data, bins=bins, range=x_range, density=density
```

Comment:
Why not skip the `if` `else` stuff and instead just pass in `normalize`?
```suggestion
        a=data, bins=bins, range=x_range, density=normalize
```

Reply:
Good point..not sure what I was thinking there! Also, thanks for stopping by :)